.circleci/config.yml

```diff
@@ -422,12 +422,10 @@ jobs:
       - install-dependencies
       - run:
           name: Run tests
-          # StoreKit tests currently get stuck.
-          # See https://discuss.circleci.com/t/severe-performance-problems-with-xcode-15/49205/11?u=nachosoto
-          command: bundle exec fastlane test_ios skip_sk_tests:true
```

Comment:
Performance didn't improve (see https://discuss.circleci.com/t/severe-performance-problems-with-xcode-15/49205/47?u=nachosoto), but let's see if at least these tests run now.

Reply:
Narrator: still unusable. And apparently Azure has the same issue, so it's Apple's fault: https://twitter.com/manuelescrig/status/1710405648628727816?s=61&t=10tcUXXCIjxv9O6sDQeMAw